src/test/data/JsonSerializableAddressBookTest/duplicateMeetingAddressBook.json

```diff
@@ -0,0 +1,22 @@
+{
+  "persons": [ {
+    "name": "Alice Pauline",
+    "phone": "94351253",
+    "email": "alice@example.com",
+    "telegram": "al_02",
+    "tagged": [ "friends" ]
+  } ],
+  "meetings": [ {
+    "name": "Valid Meeting",
+    "date" : "12/03/2022",
+    "startTime" : "1600",
+    "endTime" : "1700",
```

Comment:
I think right now our isSameMeeting method in Meeting class defines duplicate meetings as meetings with same name and date. We can just that implementation to also require it to be same startTime and endTime if it is intended to be that way instead.